core/trino-main/src/main/java/io/trino/connector/system/GlobalSystemConnector.java

```diff
@@ -15,7 +15,7 @@
 
```

Comment:
Improve the commit message. In such case I would write `Introduce X` and then explain why X is beneficial